lightning/src/routing/scoring.rs

```diff
@@ -1055,7 +1072,8 @@ impl<G: Deref<Target = NetworkGraph<L>>, L: Deref, T: Time> Writeable for Probab
 	#[inline]
 	fn write<W: Writer>(&self, w: &mut W) -> Result<(), io::Error> {
 		write_tlv_fields!(w, {
-			(0, self.channel_liquidities, required)
+			(0, self.channel_liquidities, required),
+			(1, self.banned_nodes, vec_type),
```

Comment:
Do we need/want to serialize these? I'd imagine they can be explicitly added after parsing or passed in as an arg instead. Presumably a list of such nodes is maintained elsewhere.

Reply:
Removed serialization now with https://github.com/lightningdevkit/rust-lightning/pull/1550/commits/1382696a4a45139553565e06afe9cc7493f97243.

Reply:
I'd say keep the methods but leave it out of the config, as you have it, since its more of a dynamic setting. I guess the question would then be if it should be included in the `ReadableArgs` parameter.